x-pack/plugins/cases/public/components/recent_cases/icon_with_count.tsx

```diff
@@ -19,24 +31,27 @@ const FlexGroup = styled(EuiFlexGroup)`
 const OuterContainer = styled.span`
   width: fit-content;
 `;
-export const IconWithCount = React.memo<{ count: number; icon: string; tooltip: string }>(
-  ({ count, icon, tooltip }) => (
-    <OuterContainer>
-      <EuiToolTip content={tooltip}>
-        <FlexGroup alignItems="center" gutterSize="none">
-          <EuiFlexItem grow={false}>
-            <Icon color="subdued" size="s" type={icon} />
-          </EuiFlexItem>
+export const IconWithCount = React.memo<{
+  count: number;
+  icon: string;
+  tooltip: string;
+  color?: IconColor;
+}>(({ count, icon, tooltip, color }) => (
+  <OuterContainer>
+    <EuiToolTip content={tooltip}>
+      <FlexGroup alignItems="center" gutterSize="none">
+        <EuiFlexItem grow={false}>
+          <Icon color={color ?? 'subdued'} size="s" type={icon} />
```

Comment:
The `Icon` has a right margin of 8px. I found it a bit big. What do you think about reducing it to `4px`?